global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
     "rollForward": "latestFeature",
-    "version": "8.0.100"
+    "version": "9.0.100-rc.2.24474.11"
```

Comment:
Needs to finish effort similar to #2251 for all packages.
The biggest leftovers are Geneva, OneCollector, Examples.

There is also a lot of others cleanups to be finished. I doubt that I find time this week to move this forward.